source/clear-linux/tutorials/azure.rst

```diff
@@ -436,11 +436,11 @@ For this tutorial, we are using the |CL| Basic SKU for our VM.
 
    You are now logged into your new |CL| VM as [user], where [user] is your
    user name. To check which software bundles are included with
-   this image, run the :command:`swupd bundle-list` command:
+   this VM image, run the :command:`swupd bundle-list` command inside the VM:
```

Comment:
need to add "sudo" in front of the swupd command.  The bask example has this so this is just for consistency

Reply:
added sudo :)

Reply:
thanks!  LGTM then...ship it!